### ISSUE TITLE
Fixes for modal

### ISSUE THREAD

**Issue by mirestrepo** [brown-ccv/react-aframe-volume-renderer#26]
1. Put modal in the same flex-row as volume renderer

```
     <div>  

        <div className="d-flex flex-column">
          
            <div className="control-button my-3 mx-5">
              <Button icon="pi pi-arrow-right" label="Controls" onClick={(e) => this.setState({sideBarVisible:true})}/>
            </div>

           <div className="d-flex flex-row">
         <Sidebar modal={false} className="ui-sidebar-lg" visible={this.state.sideBarVisible} onHide={(e) => this.setState({sideBarVisible:false})}>
           <Controls/>
        </Sidebar>
            <div className="voume-renderer mx-1">
              <VolumeRenderer/>
            </div>
          </div>
        </div>
```

**Comment by fernandogelin** [comment]
It looks like the `sidebar` component has css `top: 0` which is forcing it to always be at the top. To override that, you need this:

```
.p-sidebar-left  {
    top: auto !important;
} 
```

**Comment by kmilo9999** [comment]
Fixed on commit  8b5ea37
